src/reader/TranslatableText.js

```diff
@@ -110,15 +123,17 @@ export function TranslatableText({
             pronouncing={pronouncing}
             translatedWords={translatedWords}
             setTranslatedWords={setTranslatedWords}
+            isBookmarkWord={isBookmarkWord}
           />
         );
       }
-      if (foundInstances[0] === word.id) {
+      
+      if (foundInstances[0] === word.id && exerciseType !== "Translate_What_You_Hear" && exerciseType !== "Select_L1W_fitting_L2T") {
         // If we want, we can render it according to words size.
         // "_".repeat(word.word.length) + " ";
         return  "_______ "
       }
-      if (foundInstances.includes(word.id)) {
+      if (isBookmarkWord && exerciseType !== "Translate_What_You_Hear" && exerciseType !== "Select_L1W_fitting_L2T") {
```

Comment:
I think these conditions can start to be difficult to read. Maybe define a variable that describes what `exerciseType !== "Translate_What_You_Hear" && exerciseType !== "Select_L1W_fitting_L2T"` intends to check for?